src/scss/constants.scss

```diff
@@ -66,6 +66,8 @@ $z-layers: (
   // Audience
   audience-emoji-reaction: 1000,
   audience-video: 2,
+  audience-participant-add-button: 1,
+  audience-video-container: 0,
```

Comment:
I'd suspect that most of these `z-index`s aren't required.

Reply:
Deleted the unneded one